phase1/gce/Kconfig

```diff
@@ -6,6 +6,12 @@ config phase1.gce.os_image
 	help
 	  The OS image you would like to use.
 
+config phase1.gce.os_ssh_user
```

Comment:
I definitely see this as something useful across the different phase1 providers, even if it's only used by GCE right now. What do you think about making it `phase1.ssh_user`?